### PR TITLE
Product Import

### PR DESCRIPTION
**Pull request by marcushohlbein**

### CODE REVIEW

File: services/importService.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  axios
+    .get(url)
+    .then(res => {
+      const products = res.data.results
+      products
+        .filter(product => product.retailPrice !== '' || null || undefined)
+        .filter(product => product.styleId !== '' || null || undefined)
+        .filter(product => product.media.imageUrl !== null || undefined)
+        .filter(product => product.media.smallImageUrl !== null || undefined)
+        .filter(product => product.media.thumbUrl !== null || undefined)
```
Comment (mhstrkmp) [inline services/importService.js:22]: Maybe you could refactor this. A lot of repetition in there.

**Review by mhstrkmp — COMMENTED** [review]
Good work 👍

File: services/importService.js (hunk 1 above)
Comment (marcushohlbein) [reply services/importService.js:22]: any idea? thought about it, but found no way to solve it 🤷‍♂️

Comment (mhstrkmp) [reply services/importService.js:22]: Could you show a sample of the products object which you get from your fetch?

Comment (marcushohlbein) [reply services/importService.js:22]:
```
{
      "id": "02600cb6-9369-46ca-93fb-a1ade2212ea4",
      "brand": "Jordan",
      "colorway": "Bordeaux/White-Metallic Silver",
      "gender": "men",
      "name": "Bordeaux",
      "releaseDate": "2021-10-20",
      "retailPrice": 170,
      "shoe": "Jordan 1 Retro High",
      "styleId": "555088-611",
      "title": "Jordan 1 Retro High Bordeaux",
      "year": 2021,
      "media": {
        "imageUrl": null,
        "smallImageUrl": null,
        "thumbUrl": null
      }
},
```

Comment (mhstrkmp) [reply services/importService.js:22]: > any idea? thought about it, but found no way to solve it 🤷‍♂️

Hey @marcushohlbein sorry for the late reply.
Today I found an interesting article which seems to do exactly what we were looking for 😉

https://www.freecodecamp.org/news/supercharged-sorts-in-javascript/

**Pull request merged**